docker-compose.yml

```diff
@@ -6,8 +6,14 @@ services:
       - .:/python
     working_dir: /python
     depends_on:
-      - db
+      db:
+        condition: service_healthy
   db:
     image: postgres:10
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
```

Comment:
I'm not really familiar with these files but I assume this is fine?

Reply:
Yes, this is to just check postgres is ready before running the tests